backend/src/infrastructure/database/repositories/games.repository.spec.js

```diff
@@ -122,8 +125,87 @@ describe("GamesRepository", function () {
     });
   });
 
-  describe(".getXgamesWithUncheckedPlayerHistory returns an array of games with unchecked player histories.", function () {
-    describe("If the amount of 3 is passed in, with two valid documents", function () {
+  describe(".getGamesWithoutDetails", function () {
+    describe("When 4 games without details are requested", function () {
+      beforeAll(async function () {
+        this.databaseClient = await initiateInMemoryDatabase(["games"]);
+
+        await this.databaseClient.insertMany("games", getGamesDatasetMock());
+
+        const gamesRepo = new GamesRepository(this.databaseClient);
+
+        this.result = await gamesRepo.getGamesWithoutDetails(4);
+      });
+
+      afterAll(function () {
+        this.databaseClient.disconnect();
+      });
+
+      it("four games are returned", function () {
+        expect(this.result.length).toBe(4);
+      });
+
+      it("the games are an instance of Game", function () {
+        expect(this.result[0]).toBeInstanceOf(Game);
+        expect(this.result[1]).toBeInstanceOf(Game);
+        expect(this.result[2]).toBeInstanceOf(Game);
+        expect(this.result[3]).toBeInstanceOf(Game);
+      });
+
+      it("the first game is missing the release date", function () {
+        expect(this.result[0].id).toBe(239140);
```

Comment:
I left the id's if we ever need to change/check something - it will be much easier to see which game is being found.